app/views/users/index.html.erb

```diff
@@ -1 +1,25 @@
-<h1>List of all users</h1>
\ No newline at end of file
+<h1 class='text-center mt-4'>Users</h1>
+
+<% @users.each do |user| %>
+<%= link_to user_path(id:user.id), 
+class:"nav-link mt-4 d-flex justify-content-center" do %>
+    <div class="card mb-3 text-center" style="width: 800px">
+      <div class="row g-0 text-center">
+        <div class="col-md-4">
+          <img src="<%= user.photo %>" class="img-fluid rounded-start" alt="profile">
+        </div>
+        <div class="col-md-8">
+      <div class="card-body d-flex flex-column">
+  <div class="text-start mt-5">
+    <h4 class="card-title"><%= user.name %></h4>
+  </div>
+  <div class="text-end mt-5">
+    <p class="card-text"><h6>Number of posts: <%= user.post_counter %></h6></p>
+  </div>
+</div>
+        </div>
+      </div>
+    </div> 
+
+  <%end%>
```

Comment:
- [ ] Your code structure looks fine but users are not being rendered on the screen as expected. I created and stored users using the rails console command and it went successful and expected users to show but they're not. Kindly work on that 👍🏽
- [ ] Below is what I get on the screen with just a Users text

<img width="1709" alt="Screenshot 2023-06-17 at 10 51 13 AM" src="https://github.com/Koshao-sys/blog-app/assets/50941074/30a8c4f8-983f-4d17-a56f-f2f8f71514a7">

Reply:
Hey @AmaduKamara Thanks for the review. Just checked this it loads on my end. e.g user = User.create(name: "John Doe", photo: "https://avatars.githubusercontent.com/u/92449654?v=4", bio: "Some bio text", post_counter: 0, created_at: Time.current, updated_at: Time.current)

<img width="695" alt="Screenshot 2023-06-17 142643" src="https://github.com/Koshao-sys/blog-app/assets/92449654/5feecab7-3acf-47a4-9b16-f4f43e06912f">

Reply:
- [ ] [OPTIONAL] @Koshao-sys the users render working fines, you have a minor issue with creating users, where if I omit the post_counter I get an error, since this is not part of the requirements I will mark it as optional, but I highly recommend that you add a default value to post_counter in your migrations to avoid this issue

![image](https://github.com/Koshao-sys/blog-app/assets/89282221/9619f11c-ff83-4467-947e-bf68dbd68f18)